src/Controller/CRUDController.php

```diff
@@ -1444,6 +1452,26 @@ protected function preList(Request $request)
         return $this->get('translator')->trans($id, $parameters, $domain, $locale);
     }
 
+    private function checkParentChildAssociation(Request $request, $object)
+    {
+        if (!($parentAdmin = $this->admin->getParent())) {
+            return;
+        }
+
+        $parentId = $request->get($parentAdmin->getIdParameter());
+
+        $propertyAccessor = PropertyAccess::createPropertyAccessor();
+        $propertyPath = new PropertyPath($this->admin->getParentAssociationMapping());
+
+        if ($parentAdmin->getObject($parentId) !== $propertyAccessor->getValue($object, $propertyPath)) {
```

Comment:
Could this break somehow? I mean what happens if it does not have the property? It breaks or it breaks on a previous step? Was it breaking before?

Reply:
It would break before:

```
Symfony\Component\PropertyAccess\Exception\InvalidArgumentException: The property path constructor needs a string or an instance of "Symfony\Component\PropertyAccess\PropertyPath". Got: "NULL"
```

as you don't have to set `$parentAssociationMapping`, this could be a BC break, will add if and just skip it and leave a `NEXT_MAJOR`